.github/workflows/samples.yaml

```diff
@@ -12,3 +12,12 @@ jobs:
       - name: Run checkstyle
         run: mvn -P lint --quiet --batch-mode checkstyle:check
         working-directory: samples/snippets
+  compile:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
```

Comment:
Could we do this for both Java 8 and Java 11? We had a failure in the past where a compilation error would only occur for Java 11 and not for Java 8.

Reply:
Yup, changed it to compile against Java 8 and 11.